src/styles/index.ts

```diff
@@ -5032,6 +5032,50 @@ const styles = (theme: ThemeColors) =>
             fontSize: variables.fontSizeNormal,
             fontWeight: FontUtils.fontWeight.bold,
         },
+
+        skeletonBackground: {
+            flex: 1,
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+        },
+
+        emptyStateForeground: (isSmallScreenWidth: boolean) => ({
+            justifyContent: 'center',
+            alignItems: 'center',
+            height: '100%',
+            padding: isSmallScreenWidth ? 24 : 0,
```

Comment:
Why 24? Is this `variables.componentBorderRadiusRounded`?

Reply:
It is value taken from screenshot 😄 
I tried to reproduce screenshots attached to the issue and 24px looks very similar